### DISCUSSION TITLE
Already attached virtual display disconnecting

### DISCUSSION THREAD

**Discussion by ykracht** [rustdesk/rustdesk#9366] [General]
### Bug Description

I use a Hyper-V VM which I need to have at two displays for automation purpose, it's being attached by the same virtual display software rustdesk uses (usbmmidd). When logging in to a device that has a virtual display already attached, it's being removed on disconnect.

### How to Reproduce

Have a virtual display attached before connecting, connect and disconnect. The virtual display attached before connection is removed.

### Expected Behavior

If a virtual display is connected before connecting, don't remove it when disconnecting.

### Operating system(s) on local side and remote side

Windows 11 -> Windows Server 2022 DC

### RustDesk Version(s) on local side and remote side

1.3.0 -> 1.3.0

### Screenshots

![image](https://github.com/user-attachments/assets/08b2e128-cd8f-41f4-923c-a8195ff865a1)

https://github.com/user-attachments/assets/605d5222-35a0-44f0-9731-dc63a6c3b1d2

**Comment by rustdesk** [comment]
We know this, but sorry, this virtual display software can not know which one is created by us or by others, we have to choose remove all. Before we have our virtual display, we have to keep this behavior.

**Reply by rustdesk** [reply]
@fufesou don't do removing if we do not create VD.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/9372

**Reply by ykracht** [reply]
Thanks for fixing anyway, works like a charm!
